src/test/kotlin/com/ampnet/reportservice/controller/AdminControllerTest.kt

```diff
@@ -65,7 +65,35 @@ class AdminControllerTest : ControllerTestBase() {
 
             val pdfContent = result.response.contentAsByteArray
             verifyPdfFormat(pdfContent)
-            // File(getDownloadDirectory(usersAccountsSummary)).writeBytes(pdfContent)
+            // File(getDownloadDirectory("usersAccountsSummary.pdf")).writeBytes(pdfContent)
+        }
+    }
+
+    @Test
+    @WithMockCrowdfundUser(privileges = [PrivilegeType.PRA_PROFILE])
+    fun mustBeAbleToDownloadXlsxForVerifiedUsers() {
+        suppose("User service will return a list of users") {
+            val user = createUserExtendedResponse(userUuid, role = Role.PLATFORM_MANAGER)
+            val secondUser = createUserExtendedResponse(secondUserUuid)
+            val thirdUser = createUserExtendedResponse(thirdUserUuid)
+            val coopResponse = createCoopResponse()
+            val response = createUsersExtendedResponse(
+                listOf(user, secondUser, thirdUser), coopResponse
+            )
+            Mockito.`when`(userService.getAllActiveUsers(coop))
+                .thenReturn(response)
+        }
+
+        verify("Platform manager can get xlsx report") {
+            val result = mockMvc.perform(
+                MockMvcRequestBuilders.get("/admin/report/xlsx")
+                    .param("type", XlsxType.VERIFIED.name)
+            )
+                .andExpect(MockMvcResultMatchers.status().isOk)
+                .andReturn()
+
+//            val content = result.response.contentAsByteArray
```

Comment:
Commented out on purpose?

Reply:
Yes, so we can easily generate and check the file.

Reply:
Maybe add a comment to explain why it's here? like: 
```
Uncomment to generate file locally.
```